README.md

```diff
@@ -28,31 +28,54 @@ npm install -g screepers-steamless-client
 screepers-steamless-client
 ```
 
-### Run with Docker Compose
+### Install or Run with Docker Compose
 
 Use Docker Compose to run the client.
-- Download the [`compose.yaml`](compose.yaml) file and place it in an empty folder.
-- Alternatively, you can add the `client` entry from [`compose.yaml`](compose.yaml) to an existing Docker Compose configuration (e.g., combine it with a Screeps server launcher). If you do this, make sure to use the `--internal_backend` argument in the command to reference the Screeps server container address, like this:
 
+1. Clone this repository
+
+2. Locate the Screeps `package.nw` file from the Steam installation of Screeps. If you are unsure of where to find it, you can right click the game from your Steam library and select the "Browse local files" option.
+
+3. Create a `.env` file in the repo root directory and set `SCREEPS_NW_PATH` to the path of `package.nw`:
+
+    1. If you have Screeps installed on your local machine via Steam, you can reference it directly. For example, on macOS:
+        ```bash
+        SCREEPS_NW_PATH="~/Library/Application Support/Steam/steamapps/common/Screeps/package.nw"
+        ```
+
+    2. If Steam and/or Screeps is not installed locally, you can copy `package.nw` from a remote system and place it in the `vendor` subdirectory:
```

Comment:
The old instructions promotes purchasing Screeps by only recommending it be installed through Steam. There's also logic to detect the local Steam installation path for easy setup when this proxy runs on the same system where Screeps is installed.

Suggesting the transfer of game files across different systems might inadvertently encourage sharing copies with others, which could negatively impact the developers' revenue. 

I propose revising the README to avoid mentioning the movement of game files to different systems, ensuring we support the developers by promoting legitimate purchases and installations.

Reply:
Agreed. The goal of this change is not to promote piracy, but rather to facilitate the use of the purchased game on other systems a legal purchaser owns which do not have Steam installed. I'll make some tweaks to the language to reflect this.

Reply:
@admon84 I've added language to the "Requirements" section to explicitly encourage purchasing the game via Steam. The new section also points out the benefits of deploying the proxy on a system on which the game is installed via Steam.

The Steam store link for the game is still retained at the top of the README.

The `package.nw` search logic has not been modified, and the changes to the README still retain the macOS example of the hard-coded path to the client binary from a Steam library.